tests/transactional/enable_fips.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: FIPS
+# Summary: Enable FIPS on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use Mojo::Base "consoletest";
+use testapi;
+use transactional;
+use bootloader_setup qw(change_grub_config);
+use version_utils qw(is_sle_micro);
+
+sub run {
+    select_console 'root-console';
+
+    # make sure fips is not enabled
+    assert_script_run("grep '^0\$' /proc/sys/crypto/fips_enabled");
+
+    # install fips pattern
+    record_info('INFO', "Installing FIPS pattern");
+    if (is_sle_micro) {
+        trup_call("pkg install -t pattern microos-fips");
+    } else {
+        trup_call("pkg install -t pattern fips");
```

Comment:
You can do `pkg install pattern:fips fipscheck`. But why `fipscheck`? It's not used here

Reply:
dracut wants it... I will give you a VR about a potential bug in MicroOS.

Reply:
Apparently using `fipscheck` is not allowed for whatever reason, it was explicitly removed: https://bugzilla.suse.com/show_bug.cgi?id=1198065